pages/linux/qm-cloudinit-dump.md

```diff
@@ -0,0 +1,8 @@
+# qm cloudinit dump
+
+> Get automatically generated cloudinit config.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Get automatically generated cloudinit configs:
+
+`qm cloudinit dump {{vmid}} {{type}}`
```

Comment:
What is a `meta` config? :thinking: 
```suggestion
`qm cloudinit dump {{vm_id}} {{meta|network|user}}`
```

Reply:
I think it specifies that metadata should passed to cloud logs.